### PR TITLE
Make SE wizard navigation better

### PR DESCRIPTION
**Pull request by jggoebel**
This PR will make the SE wizard navigation skippable if an SE is edited.
Also fixes small bug with advanced page when editin SEs.

Fixes https://github.com/hobbyfarm/hobbyfarm/issues/164

### CODE REVIEW

File: src/app/event/new-scheduled-event/new-scheduled-event.component.html [hunk 1]
```diff
@@ -349,7 +349,7 @@ <h3>Simple Mode Compatibility</h3>
     </ng-container>
   </clr-wizard-page>
 
-  <clr-wizard-page>
+  <clr-wizard-page [clrWizardPageNextDisabled]="(selectedcourses.length == 0 && selectedscenarios.length == 0) || selectedEnvironments.length == 0 || !vmCounts.valid || !simpleModeVmCounts.valid || (se.start_time >= se.end_time) || !se.start_time || !se.end_time || !eventDetails.valid">
```
Comment (ebauman) [inline src/app/event/new-scheduled-event/new-scheduled-event.component.html:352]: Is it possible we make this calculated in the typescript? this is sort of... big.

File: src/app/event/new-scheduled-event/new-scheduled-event.component.ts [hunk 2]
```diff
@@ -316,7 +318,9 @@ export class NewScheduledEventComponent implements OnInit {
   }
 
   ngOnChanges() {
+    console.log("onChanges ran");
```
Comment (ebauman) [inline src/app/event/new-scheduled-event/new-scheduled-event.component.ts:321]: needs removal

**Review by ebauman — CHANGES_REQUESTED** [review]

File: src/app/event/new-scheduled-event/new-scheduled-event.component.html (hunk 1 above)
Comment (ebauman) [reply src/app/event/new-scheduled-event/new-scheduled-event.component.html:352]: ideally using if/else so it's easier to understand for the next dev

File: src/app/event/new-scheduled-event/new-scheduled-event.component.ts (hunk 2 above)
Comment (jggoebel) [reply src/app/event/new-scheduled-event/new-scheduled-event.component.ts:321]: oh shoot. i will remove this. 👍

File: src/app/event/new-scheduled-event/new-scheduled-event.component.html (hunk 1 above)
Comment (jggoebel) [reply src/app/event/new-scheduled-event/new-scheduled-event.component.html:352]: Yep i will create a method for this check.

**Comment by jggoebel** [comment]
I commited the new changes

**Review by ebauman — APPROVED** [review]

**Pull request merged**